### PR TITLE
Hide votes while vote is running

### PR DESCRIPTION
**Pull request by elmaxe**
This PR does two major things

- Remove the ability for drek to see **what** alternatives people have voted on when a voting is **open** (images below)
  - For drek to see if and how many people have voted, an indicator that shows how many people have voted has been added
- Protect the `/log` path. There has been no link to this page in the navbar, however, the page has not been protected since the launch of mentometer, allowing you to navigate to [/log](https://mentometer.datasektionen.se/log) and see all results...

#### Small fixes:
- Add nodemon as a dev-dependency to the backend, this allows you to hot-reload the server while developing
  - Use `npm run dev:server` to run server with nodemon.
- Updated `README.md` according to the change above
- Automatically make the user admin when running locally.
  - If the environment variable `NODE_ENV` is set to `development`, you will automatically be given admin privileges

These images show what the voting process looks like (for drek) after the changes listed above.
### Before opening:
image
### When vote is open:
image
### When vote is open and someone has voted:
image
### When vote is closed:
image

### CODE REVIEW

File: src/Results/Results.js [hunk 1]
```diff
@@ -2,16 +2,27 @@ import React, { Component } from 'react'
 
 class Results extends Component {
 	render() {
-		const {question, alternatives, votes} = this.props
+		const { question, alternatives, votes, open, users } = this.props
+
+		const numVotes = votes.reduce((a, b) => a + b, 0);
+		// user.attending is a boolean
+		// However, true + true = 2 (and so on) in Javascript, so the map is actually unnecessary
+		const attending = users.map(u => u.attending ? 1 : 0).reduce((a, b) => a + b, 0)
```
Comment (niklasvatn) [inline src/Results/Results.js]: Skulle nog byta den till `users.filter(user => user.attending).length`

File: src/Results/Results.js [hunk 2]
```diff
@@ -2,16 +2,27 @@ import React, { Component } from 'react'
...
+
 		return (
 			<div>
 				<h1>{ question }</h1>
+				<h3>Antal som röstat: <span style={{color: "black"}}>{numVotes} / {attending}</span></h3>
 	            <ul className="results">
 	              { alternatives.map((alternative, index) => (
 	                <li key={JSON.stringify(alternative)}>
-	                  <div className="theme-color bar-label">{ alternative.content } ({ votes[index] })</div>
+	                  <div className="theme-color bar-label">{ alternative.content } { open ? "?" : votes[index] }</div>
```
Comment (niklasvatn) [inline src/Results/Results.js]: Ingen kanske nämnt det för dig. Meeen om du har ett långt svarsalternativ så kunde man tidigare inte se röstsiffrorna. 
Skulle vara najs om det går att åtgärda.

Comment (niklasvatn) [reply src/Results/Results.js]: Om du inte redan gjorde det.

File: src/Results/Results.js (hunk 1 above)
Comment (elmaxe) [reply src/Results/Results.js]: Efter `filter` kommer vi då ha en lista med `user`-objekt istället för en lista med 1:or och 0:or, men kan lösas såhär:
`users.filter(u => u.attending).reduce((a, b) => a + b.attending, 0)`

File: src/Results/Results.js (hunk 2 above)
Comment (elmaxe) [reply src/Results/Results.js]: Fixed in b4ecea8

File: src/Results/Results.js (hunk 1 above)
Comment (elmaxe) [reply src/Results/Results.js]: Fixed in b4ecea8

**Review by niklasvatn — APPROVED** [review]
Looks good!

**Pull request merged**